applications/luci-app-watchcat/luasrc/model/cbi/watchcat/watchcat.lua

```diff
@@ -1,53 +1,123 @@
+        translate(
+            "Here you can set up several checks and actions to take in the event that a host becomes unreachable. " ..
+                "Click the <b>Add</b> button at the bottom to set up more than one action."))
 
-s = m:section(TypedSection, "watchcat")
+s = m:section(TypedSection, "watchcat", "Watchcat", "These rules will govern how this device reacts to network events.")
 s.anonymous = true
 s.addremove = true
 
-mode = s:option(ListValue, "mode",
-		translate("Operating mode"))
-mode.default = "allways"
-mode:value("ping", "Reboot on internet connection lost")
-mode:value("allways", "Periodic reboot")
-
-forcedelay = s:option(Value, "forcedelay",
-		      translate("Forced reboot delay"),
-		      translate("When rebooting the system, the watchcat will trigger a soft reboot. " ..
-				"Entering a non zero value here will trigger a delayed hard reboot " ..
-				"if the soft reboot fails. Enter a number of seconds to enable, " ..
-				"use 0 to disable"))
-forcedelay.datatype = "uinteger"
-forcedelay.default = "0"
-
-period = s:option(Value, "period", 
-		  translate("Period"),
-		  translate("In periodic mode, it defines the reboot period. " ..
-			    "In internet mode, it defines the longest period of " .. 
-			    "time without internet access before a reboot is engaged." ..
-			    "Default unit is seconds, you can use the " ..
-			    "suffix 'm' for minutes, 'h' for hours or 'd' " ..
-			    "for days"))
-
-pinghost = s:option(Value, "pinghosts", 
-		    translate("Ping host"),
-		    translate("Host address to ping"))
+mode = s:option(ListValue, "mode", translate("Mode"))
+mode.default = "ping_reboot"
+mode:value("ping_reboot", translate("Ping Reboot: Reboot this device if a host is unreachable"))
+mode:value("periodic_reboot", translate("Periodic Reboot: Reboot this device periodically at a set interval of time"))
+mode:value("restart_iface", translate("Restart Interface: Restart an interface if a host is unreachable"))
+
+period = s:option(Value, "period", translate("Period"),
+             translate("In Periodic Reboot mode, it defines how often to reboot. <br>" ..
+                           "In Ping Reboot mode, it defines the longest period of " ..
+                           "time without a reply from the Host To Check before a reboot is engaged. <br>" ..
+                           "In Network Restart mode, it defines the longest period of " ..
+                           "time without a reply from the Host to Check before the mobile data connection is restarted." ..
+                           "<br><br>The default unit is seconds, without a suffix, but you can use the " ..
+                           "suffix <b>m</b> for minutes, <b>h</b> for hours or <b>d</b> " ..
+                           "for days. <br><br>Examples:<ul><li>10 seconds would be: <b>10</b> or <b>10s</b></li><li>5 minutes would be: <b>5m</b></li><li>1 hour would be: <b>1h</b></li><li>1 week would be: <b>7d</b></li><ul>"))
```

Comment:
The second translate into a new line and the splitting of the strings with `"<message>" ..` fits.
But the whole thing should be consistent in the whole file.

Reply:
Do you mean the extra long line at the end?  I've reformatted it so they are all approximately the same length.

Reply:
Also, should the translation strings include HTML tags, or should we take them out?  We could do something like this if it makes more sense:

`"<b>" .. translate("Hello") .. "</b>"`

Reply:
There is no default. Personally, I think it's better if you're not there. But everyone does that differently here in the LuCI.